### ISSUE TITLE
Move OTP input from entity to config field

### ISSUE THREAD

**Issue by ulope** [Dielee/volvo2mqtt#281]
**Is your feature request related to a problem? Please describe.**
The Volvo API seems to semi regularly require a new OTP code. 
Currently this has to be submitted by changing a text entity's value which is quite unusual and not easy to remember (since this happens very infrequently).

**Describe the solution you'd like**
Please move the OTP code input to a config field so it's possible to find intuitively in the UI.

**Describe alternatives you've considered**
Detect that an OTP code is needed and ask for it via a notification (I don't know enough about HA internals to know if this is easily possible)

**Additional context**
n/a

**Comment by ulope** [comment]
Stale bots are the worst and should be banned

**Comment by rock3r** [comment]
I agree the add-on should at least send a notification when there is an OTP required. You can probably copy what the Tado integration does, since they seem to have a similar issue with the APIs they call:
 1. Raise a configuration error in settings when the auth expires
 2. Implement a fix flow that sends the OTP via email, and contains a text field for the OTP
 3. When the user gives you the OTP, check it, and if successful mark the issue as solved; if not, repeat step 2

The relevant part of the Tado integration is here: https://github.com/home-assistant/core/blob/dev/homeassistant/components/tado/config_flow.py
I don't have HA development experience (and to be honest, much spare time lately), or I'd try creating a PR myself :)

**Comment by Dielee** [comment]
As this is not a "full" HA integration, your suggestion will not work. 
This integration will not get any new features, it is as it is and will only be maintained.

**Comment by rock3r** [comment]
Fair enough, I was not aware that this repository is in maintenance mode.
